newIDE/app/src/MainFrame/Onboarding/OnboardingDialog.js

```diff
@@ -27,51 +23,75 @@ const styles = {
   },
 };
 
+const USERFLOW_DEV_ID = 'ct_y5qogyfo6zbahjejcbo3dybnta';
+const USERFLOW_PROD_ID = 'ct_paaz6o2t2bhlrlyi7a3toojn7e';
+
+const userFlowIds = {
+  fr_FR: '761ac722-d74e-45e3-a870-cd0f60f3299f',
+  pt_BR: 'a31fa85f-df42-40b5-aa80-3c6a59ad9666',
+  pt_PT: 'a31fa85f-df42-40b5-aa80-3c6a59ad9666',
+  es_ES: '146d656b-900a-4294-86a5-70f90f12c1eb',
+  en: 'b1611206-2fae-41ac-b08c-0f8ad72d8c39',
+};
+
 const onboardingText = `
 In 5 minutes, you will have:
   - Created a game
   - Learned the fundamentals of GDevelop
 
-(🇬🇧 The tour is only available in English)
+(The tour is only available in 4 languages: 🇬🇧/🇺🇸 English, 🇫🇷 French, 🇪🇸 Spanish, 🇵🇹 Portuguese)
+
 We highly recommend it!
 `;
 
-const OnboardingDialog = () => {
-  const [open, setOpen] = React.useState(false);
+type Props = {|
+  open: boolean,
+  onClose: () => void,
+  onUserflowRunningUpdate: () => void,
+|};
+
+const OnboardingDialog = ({
+  open,
+  onClose,
+  onUserflowRunningUpdate,
+}: Props) => {
   const [loading, setLoading] = React.useState(false);
+  const { values } = React.useContext(PreferencesContext);
```

Comment:
`const { values: userPreferences } = React.useContext(PreferencesContext);` for better readability?